keycloak/keycloak-operator
# Intly 3225

## JIRA ID

INTLY-3225

## Additional Information

Creates an Ingress or Route if `spec.externalAccess.enabled` is set to True in the keycloak CR. A Route is only created on OpenShift. Also some refactoring.

## Verification Steps

1. Deploy keycloak from the provided example
1. Make sure a Route gets created and that it works
1. You should be able to use the route to login to the admin console using the credentials in the admin secret.
